templates/types/streaming/fastapi/app/api/routers/chat.py

```diff
@@ -14,10 +12,7 @@
     Result,
     SourceNodes,
 )
-from app.api.routers.vercel_response import (
-    ChatEngineVercelStreamResponse,
-    WorkflowVercelStreamResponse,
-)
+from app.api.routers.vercel_response import VercelStreamResponse
```

Comment:
@leehuwuj isn't that called `ChatEngineVercelStreamResponse` now? but i would actually keep the old name

Reply:
Thanks! I switched it to just use VercelStreamResponse. This way, we avoid any naming confusion in the code 😄